src/App/App.js

```diff
@@ -37,20 +38,26 @@ class App extends React.Component {
   state = {
     authed: false,
     pendingUser: true,
+    // uid: '',
```

Comment:
remove these comments, since you're not going down this path

Reply:
removed